### PR TITLE
Fix some typos and wording.

### PR DESCRIPTION
**Pull request by dhenrich**
Just some clean up around the wording and fix some typos.

### CODE REVIEW

**Comment by stgraber** [comment]
@dhenrich Looks good to me except for the missing DCO line in the commit message.

Can you add that "Signed-off-by: ..." line to your commit message?

**Comment by dhenrich** [comment]
Should be good now.

File: doc/clustering.md [hunk 1]
```diff
@@ -36,15 +37,15 @@ available on your network.
...
-clear them up before joining, since any existing data on them will be
-lost.
-
-Run `lxd init` and answer `yes` to the question about whether to use
-clustering. Pick a new node name which must be different from the one
-of the bootstrap node or any other nodes you joined so far. Then pick
-an IP or DNS address for the node and answer `yes` to the question
-about whether you're joining an existing cluster. Pick an address of
-an existing node in the cluster and check the fingerprint that gets
+clear their contents them up before joining, since any existing data
```
Comment (freeekanayaka) [inline doc/clustering.md]: I think this is not grammatically correct: "clear their contents them up". Should probably be "clear their contents up".

Comment (dhenrich) [reply doc/clustering.md]: Agreed, thanks.

**Pull request merged**